RecoCTPPS/TotemRPLocal/plugins/CTPPSDiamondLocalTrackFitter.cc

```diff
@@ -68,10 +68,7 @@ CTPPSDiamondLocalTrackFitter::produce( edm::Event& iEvent, const edm::EventSetup
   // workaround to retrieve the detset for 4-5 without losing the reference
   edm::DetSet<CTPPSDiamondLocalTrack>& tracks45 = pOut->operator[]( id_45 );
 
-  // first remove all hits from previous event
-  trk_algo_45_.clear();
-  trk_algo_56_.clear();
```

Comment:
are the algorithms created "clear" so that it's safe to have the ```clear()``` call only at the end?